lib/schemeregistrar.js

```diff
@@ -99,7 +99,7 @@ export class SchemeRegistrar extends ExtendTruffleContract(
        * true if the given scheme is able to register/unregister/modify schemes.
        *
        * isRegistering should only be supplied when schemeName is not given (and thus the scheme is non-Arc).
-       * Otherwise we determine it's value based on scheme and schemeName.
+       * Otherwise we determine it"s value based on scheme and schemeName.
```

Comment:
it"s
typo?  (it's)

Reply:
probably when I removed all single quotes because of our linter

Reply:
resolved